### DISCUSSION TITLE
ライセンスについて

### DISCUSSION THREAD

**Discussion by hideojoho** [ir100/ir100#4] [General]
## ゴール
- 本レポジトリおよび成果物のライセンスを決定する

## 動機
- 有志のみなさんが貢献しやすい、コミュニティが活用しやすいコンテンツにしたい

## 参考URL
- https://docs.github.com/ja/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository#further-reading

**Comment by pakio** [comment]
[自然言語処理100本ノックのライセンス](https://github.com/nlp100/nlp100.github.io/blob/develop/LICENSE)もMITですし、利用を広く認めつつもその責任はユーザで有ることを明記できるため、MITライセンスはいかがでしょうか？
